docs/modules/jenkins-shared-library/partials/odsComponentStageRolloutOpenShiftDeployment.adoc

```diff
@@ -158,6 +171,10 @@ _boolean_
 
 |===
 
-All the created or updated OpenShift resources are automatically labeled to ease their management.
+== Differences between tailor and helm deployments
+
+When tailor does the rollout, all the created or updated OpenShift resources are automatically labeled to ease their management.
+This is in contrast to helm rollouts which rely on the chart providing the desired labels.
```

Comment:
This might be problematic, since we rely on certain labels to able to correlate resources with their respective systems, e.g. with Aqua. FYI @maksymmaksym